test/functional/test-sanitizer.js

```diff
@@ -250,6 +250,26 @@ function runSanitizerTests() {
       expectEqualNodeLists(actual, expected);
     });
 
+    it('should output "layout" attribute', () => {
+      const img = '<amp-img layout="responsive"></amp-img>';
+      expect(sanitizeHtml(img)).to.equal(img);
+    });
+
+    it('should output "media" attribute', () => {
```

Comment:
huh, curious how `media` is working without it being whitelisted. There is a standard `media` attribute but only for `style` and `link`. Might be worth listing it explicitly.

Reply:
Both Caja and DOMPurify whitelist it broadly. DOMPurify doesn't have tag-specific whitelists to begin with, which is what helps keep its size small.